feature_dists_metric:
  path: "data/platform_dists_msc_platform.txt"

services:
  AT_node_activation_name: activate_AT_tracking
